extensions/jupiter-aggregator/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Solana Price Changelog
```

Comment:
Should we change the title to Jupiter Aggregator?

Reply:
Good catch! Thank you!